packages/runtime-js/src/main/kotlin/elide/runtime/plugins/js/JavaScript.kt

```diff
@@ -92,6 +92,17 @@ import elide.runtime.plugins.js.JavaScriptVersion.*
       "js.v8-compat" to config.v8,
     )
 
+    if(config.wasm) {
+      enableOptions(
+        "wasm.Memory64",
+        "wasm.MultiValue",
+        "wasm.UseUnsafeMemory",
+        "wasm.BulkMemoryAndRefTypes",
```

Comment:
i actually think we should eventually `Symbol<*>`-ize these like intrinsics, and potentially provide a way outside the binary to modify them or turn on experimental options. for later, though